### PR TITLE
v2v: add TC 'ESX windows poweroff external'

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check bz#1567763

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.negative_test.windows.external_poweroff.esx_67.rhev.rhv.it_default: PASS (787.27 s)

**Comment by vwu-vera** [comment]
@xiaodwan please help to review. Thanks.

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -185,6 +185,13 @@
                     checkpoint = 'rhsrvany_checksum'
                     main_vm = VM_NAME_WIN2019_V2V_EXAMPLE
                     os_version = 'win2019'
+                - external_poweroff:
+                    only esx_67
+                    only negative_test
+                    version_required = "[libguestfs-1.38.1-1.el7,)"
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:191]: The 'version_required' is only necessary when a higher version includes the fix but a lower version which is still in testing lifecyle doesn't have the fix. It's usually useful to new cases.
This case is quite old and all the main releases under testing have higher libguestfs versions, so you could delete the 'version_require' setting.

Comment (vwu-vera) [reply v2v/tests/cfg/function_test_esx.cfg]: Thanks for the information. Will pay more attention in future case. :) 

Has modified accordingly.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**